tests/acceptance/builds/prioritize-test.js

```diff
@@ -0,0 +1,84 @@
+import { module, test } from 'qunit';
+import { visit, click, triggerKeyEvent } from '@ember/test-helpers';
+import { setupApplicationTest } from 'travis/tests/helpers/setup-application-test';
+import signInUser from 'travis/tests/helpers/sign-in-user';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+module('Acceptance | builds/prioritize', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(function () {
+    const currentUser = this.server.create('user');
+    signInUser(currentUser);
+  });
+
+  test('prioritizing build button does not exsist if the build is not having permission to prioritize becuase of the organization', async function (assert) {
+    let repository =  this.server.create('repository', { slug: 'travis-ci/travis-web' });
+    let branch = this.server.create('branch', { repository, name: 'acceptance-tests', default_branch: true });
+    let gitUser = this.server.create('git-user', { name: 'Mr T' });
+    let commit = this.server.create('commit', { author: gitUser, committer: gitUser, committer_name: 'Mr T', message: 'This is a message' });
+    let build = this.server.create('build', { number: '5', state: 'queued', permissions: { 'prioritize': false }, repository, commit, branch, priority: true, });
```

Comment:
As far as I understand, the button should not appear neither when `priority: true` nor `permissions.prioritize: false`, right? If so, this test won't actually test anything, because the button will never appear as build is already high priority, independently of the permissions.

Reply:
Making the build not yet prioritized, so that it _could_ be prioritized:
```suggestion
    let build = this.server.create('build', { number: '5', state: 'queued', permissions: { 'prioritize': false }, repository, commit, branch, priority: false, });
```